packages/input/src/ux-input.html

```diff
@@ -4,10 +4,9 @@
   readonly.bind="readonly & booleanAttr"
   aria-disabled.bind="disabled & booleanAttr"
   aria-readonly.bind="readonly & booleanAttr"
-  click.trigger="focus()">
+  click.trigger="focus() || true">
```

Comment:
@MaximBalaganskiy i moved return true here so that our vm `focus()` method can stay aligned with common expectation with that method